pom.xml

```diff
@@ -36,43 +105,39 @@
                     </descriptorRefs>
                 </configuration>
             </plugin>
+
+            <!-- allows 'mvn release' command to create a new release of the project -->
```

Comment:
Do we have documentation somewhere that describes how to do releases this way?

Reply:
Standard documentation for the maven release plugin is here: https://maven.apache.org/maven-release/maven-release-plugin/

Reply:
I can embed the URL in the comment just so it's easy to lookup how to use the command and for anyone new to the project and maven.

Reply:
Done: https://github.com/reactome/release-biomodels/pull/1/commits/3410122180d2c40637f273a34ae9078efec1387f